presto-main/src/main/java/io/prestosql/execution/GrantTask.java

```diff
@@ -53,15 +58,34 @@ public String getName()
     @Override
     public ListenableFuture<?> execute(Grant statement, TransactionManager transactionManager, Metadata metadata, AccessControl accessControl, QueryStateMachine stateMachine, List<Expression> parameters)
```

Comment:
Why not to use this method in tests like it is used in `io.prestosql.execution.TestCallTask`?

Reply:
CallTask uses stateMachine argument for something more than getSession method. This one (and most other tested tasks) do not.